src/player_display.cpp

```diff
@@ -875,9 +864,14 @@ static void draw_skills_info( const catacurses::window &w_info, unsigned int lin
     werase( w_info );
 
     if( selectedSkill ) {
+        const SkillLevel &level = get_avatar().get_skill_level_object( selectedSkill->ident() );
+        std::string info_text = selectedSkill->description();
+        if( level.isRusting() ) {
+            info_text = string_format( _( "%s\n\nTheoretical level: %d (%d%%)" ), info_text,
```

Comment:
```suggestion
            info_text = string_format( _( "%s\n\nKnowledge level: %d (%d%%)" ), info_text,
```
The theoretical term was dropped in favour of "knowledge".